frontend/src/components/movies/MovieDetails.js

```diff
@@ -8,271 +8,222 @@ import { popupNotification } from '../../lib/notification'
 
 import { FaHeart } from 'react-icons/fa'
 
-class MovieDetails extends React.Component {
-  state = {
-    movie: null,
-    formdata: {
-      text: '',
-      rating: ''
-    },
-    liked_movies: '',
-    currentUserId: '',
-    liked_by: '', 
-    heartColor: 'grey',
-    liked: false,
-    commentErrors: {}
-  }
-  
+const MovieDetails = (props) => {
+  const { match } = props
```

Comment:
Probably avoid `match` being passed through via props. Instead consider using the `useParams` or `useMatch` hook here instead.